imagemounter/volume.py

```diff
@@ -558,8 +558,10 @@ def determine_fs_type(self):
                     self.fstype = 'ext'
                 elif 'bsd' in fsdesc or 'ufs 2' in fsdesc:
                     self.fstype = 'ufs'
-                elif '0x07' in fsdesc or 'ntfs' in fsdesc:
-                    self.fstype = 'ntfs'
+                elif '0x07' in fsdesc or 'ntfs' in fsdesc or 'exfat' in fsdesc:
+                    # Use blkid to differentiate between ntfs and exfat
+                    self.fstype = self._get_blkid_type()
```

Comment:
This is not the way to go, especially as the blkid call would be the next in determining the partition type anyway. This is what the `last_resort` is for. Perhaps something like this?

```python
                elif '0x07' in fsdesc or 'ntfs / exfat' in fsdesc:
                    last_resort = 'ntfs'
                    continue
                elif 'ntfs' in fsdesc:
                    self.fstype = 'ntfs'
                elif 'exfat' in fsdesc:
                    self.fstype = 'exfat'
```